### PR TITLE
Issue 130 rename waypoint to favourite

### PR DESCRIPTION
**Pull request by W0lfw00d**
As requested in issue #130 I rename 'waypoint(ing)' to 'favourite(-ing)' in the visible text as well as in the code. 

I hope is what you wanted. :)

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ export default {
   VIEWED_ASSIGNMENT_LIST:           'VIEWED_ASSIGNMENT_LIST',
   VIEWED_MAP:                       'VIEWED_MAP',
 
-  RANK_NODE_WAYPOINT:               'RANK_NODE_WAYPOINT',
+  RANK_NODE_FAVOURITE:               'RANK_NODE_FAVOURITE',
```
Comment (glsignal) [inline src/constants.js]: These values should be in vertical alignment

**Comment by W0lfw00d** [comment]
It should be aligned now. :)

**Comment by glsignal** [comment]
Perfect! Thanks :D

**Comment by mattkennedynz** [comment]
Woohoo! Thanks so much @W0lfw00d, you're officially our first community contributor 🌟 🎉 🙌 

(I'm probably breaking all the rules by asking this here, but...) What was your experience like with getting your development environment set-up, through to submitting this PR?

**Comment by W0lfw00d** [comment]
You're welcome. :bowtie:

Not sure what those rules are that your talking about. :sweat_smile: But getting set-up was pretty easy with the [Getting-Started guide](https://github.com/twingl/trailblazer-wash/wiki/Getting-Started). The only thing that throw me off at first was: 
> Visit chrome://extensions and turn on "developer mode" - this will allow you to load what's called an unpacked extension; the code you've just built.

Which I thought to mean loading the `build/` folder. I figured it after a while out because the manifest file was in the root folder. Otherwise it was easy to follow.

**Comment by mattkennedynz** [comment]
Ah, the rules I was referring to is that my question didn't strictly relate to this PR. But I jumped on the opportunity anyway 😉.

Hey thanks a lot for that feedback! I'll make a note to clarify that section specifically. It's great to have had you run through that with fresh eyes.

Go well, @W0lfw00d! 🚀

**Pull request merged**